src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java

```diff
@@ -11087,6 +11100,55 @@ public static Value voting_booth_initiatives(
     return value;
   }
 
+  public static Value beret_busking_effects(ScriptRuntime controller) {
+    var power = KoLCharacter.getTotalPower();
+    var cast = Preferences.getInteger("_beretBuskingUses");
+    return beret_busking_effects(controller, new Value(power), new Value(cast));
+  }
+
+  public static Value beret_busking_effects(
+      ScriptRuntime controller, final Value power, final Value cast) {
+    MapValue value = new MapValue(DataTypes.EFFECT_TO_INT_TYPE);
+
+    // Calculate the capped power
+    var cappedPower =
+        Math.min(power.contentLong, 1100)
+            + (long) Math.floor(Math.pow(Math.max(0, power.contentLong - 1100), 0.8));
+
+    // $effect[none] will indicate the meat gained
```

Comment:
I think this is an inherently confusing way for this to work. Meat gain should probably be another function, or this should return an object, or something other than "the return value is effects mapped to duration, except that if the effect is none it's meat gained instead".